apps/src/code-studio/progressRedux.js

```diff
@@ -40,8 +40,9 @@ const OVERWRITE_RESULTS = 'progress/OVERWRITE_RESULTS';
 const PEER_REVIEW_ID = -1;
 
 const initialState = {
-  // These first fields never change after initialization
   currentLevelId: null,
+
+  // These first fields never change after initialization
```

Comment:
This is an interesting comment 🙂 seems like a good reason to pull these values out of this store at some point...